src/interfaces/IPot.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.8.0;
+
+interface IPot {
+    function dsr() external view returns (uint256);
+    function chi() external view returns (uint256);
+    function rho() external view returns (uint256);
+}
```

Comment:
```suggestion
interface IPot {

    function dsr() external view returns (uint256);
    function chi() external view returns (uint256);
    function rho() external view returns (uint256);
    
}
```

Reply:
Does this match anywhere else? Usually no new newlines I've seen with non-natspec interfaces.